docs/reference/ml/apis/dfanalyticsresources.asciidoc

```diff
@@ -0,0 +1,49 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-dfanalytics-resources]]
+=== {dfanalytics-cap} resources
+
+A {dfanalytics} configuration object has the following properties:
+
+`analysis`::
+  (object) The type of analysis that is performed on the `source`. For example: 
+  `outlier_detection`.
+
+`dest`::
+  (object) The destination configuration of the analysis, consisting of `index` 
+  and optionally `results_field` (`ml` by default). 
+
+`id`::
+  (string) The unique identifier for the {dfanalytics-job}. This identifier can 
+  contain lowercase alphanumeric characters (a-z and 0-9), hyphens, and 
+  underscores. It must start and end with alphanumeric characters. This property 
+  is informational; you cannot change the identifier for existing jobs.
+
+`source`::
+  (object) The source configuration, consisting of `index` and optionally a 
+  `query`.
+
+
+[float]
+[[oldetection-resources]]
+==== {oldetection-cap} Configuration Objects 
+
+An {oldetection} configuration object has the following properties:
+
+`n_neighbors` (Optional)::
+  (integer) Defines the value for up to how many neighbors 
```

Comment:
When this is set it defines the exact value of how many neighbors we'll use, not `up to`.